tests/cover/test_sampled_from.py

```diff
@@ -18,10 +18,19 @@
 from __future__ import division, print_function, absolute_import
 
 from hypothesis import given, settings
-from hypothesis.strategies import sampled_from
+from hypothesis.errors import HypothesisDeprecationWarning
+from tests.common.utils import fails_with
+from hypothesis.strategies import sets, integers, sampled_from
 
 
 @given(sampled_from((1, 2)))
 @settings(min_satisfying_examples=10)
 def test_can_handle_sampling_from_fewer_than_min_satisfying(v):
     pass
+
+
+@fails_with(HypothesisDeprecationWarning)
+@given(sets(integers()) | sets(integers()).map(iter))
+@settings(strict=True)
+def test_cant_sample_sets_or_iterators(s):
```

Comment:
Also not a request for changes but I twitch slightly at the use of cant in a method name because of the lack of apostrophe. I tend to use cannot.